assets/i18n/es.po

```diff
@@ -0,0 +1,720 @@
+msgid "Show advanced settings"
+msgstr "Mostrar ajustes avanzados"
+
+#: assets/page/ConversationAdd.svelte:90
+msgid "Showing %1 of %2 conversations, but the list is still loading."
+msgstr "Mostrando %1 de %2 conversaciones, pero la lista aún está cargando."
+
+#: assets/page/ConversationAdd.svelte:88
+msgid "Showing %1 of %2 conversations."
+msgstr "Mostrando %1 de %2 conversaciones."
+
+#: assets/page/Login.svelte:111
+msgid "Sign in"
+msgstr "Iniciar sesión"
+
+#: assets/page/Login.svelte:88
+msgid "Sign up"
+msgstr "Registrar"
+
+#: assets/page/Help.svelte:70
+msgid "Source code"
+msgstr "Código fuente"
+
+#: assets/page/Chat.svelte:159
+msgid "Start video conference"
+msgstr "Iniciar video conferencia"
+
+#: assets/page/Fallback.svelte:29
+msgid "Started Convos."
+msgstr "Convos ha iniciado."
+
+#: assets/page/Chat.svelte:170
+msgid "Started chatting on %1"
+msgstr "Se inició conversación en %1"
+
+#: assets/page/Help.svelte:41
+msgid "Text formatting"
+msgstr "Formato de texto"
+
+#: assets/page/Fallback.svelte:37
+msgid "The Convos Team have been searching and searching, but the requested page could not be found."
+msgstr "El Equipo de Convos ha estado buscando y buscando, pero la página solicitada no se pudo encontrar."
+
+#: assets/page/Help.svelte:33
+msgid "The following rules apply when typing a message:"
+msgstr "Las siguientes reglas aplican al escribir un mensaje:"
+
+#: assets/page/Login.svelte:51
+msgid "The link has expired."
+msgstr "El enlance ha expirado."
+
+#: assets/page/Login.svelte:51
+msgid "The link is invalid."
+msgstr "El enlace es inválido."
+
+#: assets/page/SettingsAdmin.svelte:95
+msgid "The path part is the default channel to join. \"%%23convos\" means \"#convos\"."
+msgstr "La parte de la ruta es el canal a unirse por defecto. \"%%23convos\" significa \"#convos\"."
+
+#: assets/page/SettingsAccount.svelte:92
+msgid "Theme"
+msgstr "Tema"
+
+#: assets/page/SettingsAdmin.svelte:88
+msgid "This email can be used by users to get in touch with the Convos admin."
+msgstr "Este correo electrónico puede ser utilizado ppara contactarse con el administrador de Convos."
+
+#: assets/page/SettingsAdmin.svelte:94
+msgid "This is the default connection new users will connect to."
+msgstr "Esta es la conexión por defecto a la que nuevos usuarios se conectarán."
+
+#: assets/page/SettingsAdmin.svelte:111
+msgid "This should point to a %1 instance."
+msgstr "Esto debería apuntar a la instancia %1."
+
+#: assets/page/SettingsAdminUsers.svelte:102
+msgid "This will permanently remove user settings, chat logs, uploaded files and other user related data."
+msgstr "Esto va a eliminar permanentemente los ajustes del usuario, historial de chat, archivos subidos y otros datos relacionados con el usuario."
+
+#: assets/page/SettingsAdmin.svelte:102
+msgid "Tick this box if you want to prevent users from creating custom connections."
+msgstr "Marca esta casila si deseas prevenir que los usuarios creen conexiones personalizadas."
```

Comment:
Typo: `s/casila/casilla/`

I'd probably also change "prevenir" into "evitar", but I'm not 100% sure why. :thinking: